web/ViewModels/LandlordListModel.cs

```diff
@@ -3,5 +3,6 @@ namespace BricksAndHearts.ViewModels;
 
 public class LandlordListModel
 {
-    public List<LandlordDbModel>? LandlordDisplayList { get; set; }
+    public List<LandlordDbModel>? LandlordList { get; set; }

```

Comment:
Does this need to be nullable?

Reply:
I believe so, because there could (theoretically) be a scenario in which there were no landlords.